src/fragments/lib/interactions/js/getting-started.mdx

```diff
@@ -36,8 +38,8 @@ Upon successful execution of the push command, a configuration file called `aws-
 > If your Interactions resources were created with Amplify CLI version 1.6.4 and below, you will need to manually update your project to avoid Node.js runtime issues with AWS Lambda. [Read more](/cli/migration/lambda-node-version-update)
 
 ## Manual setup
```

Comment:
I think we need to highlight how to use Lex in a slightly better way. First of all - we need to introduce to developers that you can add interactions either through the CLI, or by configuring everything manually inside `amplify.configure()`. Then, we need to mention that there is a limitation on the CLI creating Lex V1 rather than V2, and that if V2 is needed, they should use the manual setup step. 

I think introducing manual setup and the cli before introducing these concepts is a bit confusing for a new developer using Amplify.

Reply:
That is a good point let me think on that

Reply:
I will address this at a later time. Let us get this merged for now.